server/app/main.py

```diff
@@ -31,17 +35,18 @@
 
 dict_of_art_ids = {}
 dict_of_exe_ids = {}
-
```

Comment:
In future for locking, we might need to use database. If we have multiple replicas of the app server this logic would fail. Maybe we can just add this as comment for reference.